rackhd_BIST/rackhd_config_template.json

```diff
@@ -0,0 +1,58 @@
+{
+    "required": {
```

Comment:
Create a JSON-schema file for the rackhd config file will be much more easier to validate the type of each entry. Of course, this config file even can move into RackHD source code.